lib/pages/home_tab.dart

```diff
@@ -277,32 +277,31 @@ class InfiniteHomeList extends HookWidget {
           listingType != PostListingType.community, 'only subscribed or all');
 
       final instances = () {
-        if (listingType == PostListingType.all) {
-          return accStore.instances;
-        } else {
+        if (listingType == PostListingType.subscribed) {
```

Comment:
@krawieck Was there a reason why the `if` was the other way around? I think we want `loggedInInstances` only when Subscribed is selected, no?

Reply:
it was that way cuz there were only 2 possible values that listingType can have, so it didn't really matter